sensor/common/detector/metrics/metrics.go

```diff
@@ -15,13 +16,51 @@ var (
 		Help:      "Time spent in exponential backoff for the ImageScanInternal endpoint",
 		Buckets:   prometheus.ExponentialBuckets(4, 2, 8),
 	})
+	networkPoliciesStored = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "num_network_policies_in_store",
+		Help:      "Number of network policies (per namespace) currently stored in the in-memory store in sensor.",
```

Comment:
Nit: `currently stored in the in-memory store` feels a bit duplicate-y.
How about `currently present/saved in sensors in-memory store`?

Reply:
Rephrased in  6421fc0eb